### PR TITLE
Sync styling

### PR DESCRIPTION
**Pull request by yuinchien**

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -75,37 +82,38 @@ Directives customize how values are handled, allowing for asynchronous values, e
...
-<img width="70" height="70" src="/images/browsers/chrome_128x128.png" alt="chrome logo">
-<img width="70" height="70" src="/images/browsers/firefox_128x128.png" alt="firefox logo">
-<img width="70" height="70" src="/images/browsers/internet-explorer_128x128.png" alt="internet explorer logo">
-<img width="70" height="70" src="/images/browsers/edge_128x128.png" alt="edge logo">
-<img width="70" height="70" src="/images/browsers/safari_128x128.png" alt="safari logo">
-<img width="70" height="70" src="/images/browsers/opera_128x128.png" alt="opera logo">
+<div class="wrapper">
+
+<h1 class="title">Browser Compatibility</h1>
+<h2 class="description">PWA Starter Kit works in all major browsers (Chrome, Firefox, IE, Edge, Safari, and Opera). </h2>
```
Comment (frankiefu) [inline docs/index.md]: s/PWA Starter Kit/lit-html

**Comment by yuinchien** [comment]
@frankiefu name fixed in browser compatibility.

**Comment by yuinchien** [comment]
@arthurevans can you review this PR and let me know if things look ok? Many thanks!

File: docs/_layouts/default.html [hunk 2]
```diff
@@ -19,14 +19,20 @@
...
-      <div class="attribution">
-        Brought to you by <a href="https://www.polymer-project.org">The Polymer Project</a>.<br>
-        Copyright 2018 The Polymer Project Authors. Code licensed under the
-        <a target="_blank" href="http://polymer.github.io/LICENSE.txt">BSD License</a>. 
-        Documentation licensed under CC BY 3.0.
+      <div class="wrapper">
+        <div class="social-links">
+          <a target="_blank" href="https://github.com/Polymer/lit-html" rel="noopener" aria-label="Github"><img src="/images/logos/logo_github.svg" /></a>
+          <a target="_blank" href="https://twitter.com/polymer" rel="noopener" aria-label="Twitter"><img src="/images/logos/logo_twitter.svg" /></a>
+          <a target="_blank" href="https://groups.google.com//forum/#!forum/polymer-dev" rel="noopener" aria-label="Email"><img src="/images/logos/logo_email.svg" /></a>
```
Comment (stramel) [inline docs/_layouts/default.html]: double slash?

File: docs/index.md [hunk 3]
```diff
@@ -75,37 +82,38 @@ Directives customize how values are handled, allowing for asynchronous values, e
...
-<img width="70" height="70" src="/images/browsers/internet-explorer_128x128.png" alt="internet explorer logo">
-<img width="70" height="70" src="/images/browsers/edge_128x128.png" alt="edge logo">
-<img width="70" height="70" src="/images/browsers/safari_128x128.png" alt="safari logo">
-<img width="70" height="70" src="/images/browsers/opera_128x128.png" alt="opera logo">
+<div class="wrapper">
+
+<h1 class="title">Browser Compatibility</h1>
+<h2 class="description">lit-html works in all major browsers (Chrome, Firefox, IE, Edge, Safari, and Opera). </h2>
+<div id="browser-thumbnails" style="margin-bottom: 20px;">
+<img width="56" width="56" src="{{ site.baseurl }}/images/browsers/chrome_128x128.png" alt="chrome logo">
```
Comment (stramel) [inline docs/index.md]: Should the browser names be capitalized in the `alt`?

Comment (yuinchien) [reply docs/index.md]: Updated alt label accordingly.

File: docs/_layouts/default.html (hunk 2 above)
Comment (yuinchien) [reply docs/_layouts/default.html]: Fixed.

**Comment by stramel** [comment]
😅 that's what I was originally talking about but was unclear

**Comment by arthurevans** [comment]
Also: CI build isn't passing because of code formatting, but this doesn't appear to be the problem, so I'm going to go ahead and merge.

**Pull request merged**